### PR TITLE
add maui sample link to readme

### PR DESCRIPTION
**Pull request by jingliancui**
so that user can easy to find how to integrate with maui

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ This repository contains samples demonstrating **how to use [OpenIddict](https:/
   - [Velusia](samples/Velusia): authorization code flow demo, with an ASP.NET Core application acting as the client.
   - [Weytta](samples/Weytta): authorization code flow with Integrated Windows Authentication support and a .NET console acting as the client.
   - [Zirku](samples/Zirku): authorization code flow demo using minimal APIs with 2 hard-coded user identities, a .NET console and a SPA acting as the clients and two API projects using introspection (Api1) and local validation (Api2).
+  - [Sandbox](https://github.com/kevinchalet/openiddict-core/tree/maui_winui_sample/sandbox): .net maui sample.
```
Comment (kevinchalet) [inline README.md]: Can you please move that to the `External samples` section and change the description to `experimental MAUI (WinUI-only) sample`?

```
- **[Experimental MAUI (WinUI-only) sample](https://github.com/kevinchalet/openiddict-core/tree/maui_winui_sample/sandbox/OpenIddict.Sandbox.Maui.Client)** by [Kévin Chalet](https://github.com/kevinchalet)
```

**Review by kevinchalet — COMMENTED** [review]
Thanks for your PR. Given that the MAUI sample is an experimental sample that only supports WinUI 3, I'd prefer listing it in the external samples category.

**Comment by kevinchalet** [comment]
FYI, I just rebased the `maui_winui_sample` branch and updated the sample to use the new `ChallengeInteractivelyAsync()`/`AuthenticateInteractivelyAsync()` APIs introduced in OpenIddict 4.5.0.

File: README.md (hunk 1 above)
Comment (jingliancui) [reply README.md]: Sure, let me update the pr tonight.

Comment (jingliancui) [reply README.md]: done

**Comment by kevinchalet** [comment]
Merged. Congrats for your first contribution to OpenIddict 🎉

**Pull request merged**